.github/issue_template/custom.md

```diff
@@ -0,0 +1,16 @@
+<!---
+This is the repository and issue tracker for https://www.python.org website.
+
+If you're looking to file an issue with CPython itself, please go to
+https://bugs.python.org
+
+Issues related to Python's documentation (https://docs.python.org) can
+also be filed in https://bugs.python.org, by selecting the "Documentation" component.
+--->
+
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```

Comment:
Do we need this template to fallback to the regular ["new issue"](https://github.com/python/pythondotorg/issues/new) page? Or will GitHub show the "Don’t see your issue here? Open a regular issue." text by default even if we don't create `custom.md`?

![screenshot from 2018-07-13 02 14 13](https://user-images.githubusercontent.com/26338/42664228-42c5e154-8642-11e8-8f86-d76b1e0d2e5f.png)

Reply:
Without `custom.md` the regular issue will still show up, so I guess it's not needed :)

Reply:
I'll remove custom.